src/Assignment1/StringLogic.java

```diff
@@ -0,0 +1,94 @@
+package Assignment1;
+
+import java.util.Scanner;
+
+public class StringLogic {
+	public boolean compareString(String firstString, String secondString)
+	{
+		/*
+		 * method for comparing two strings's equality
+		 * @param firstString contains value for first string.
+		 * @param secondString contains value for second string
+		 * @return boolean true if matches else false
+		 */
+		if(firstString.length()<secondString.length() ||firstString.length()>secondString.length() )
+		{
+			return false;
+		}
+		else
+		{
+			for(int itr=0; itr<firstString.length();itr++)
+			{
+				if(firstString.charAt(itr)==secondString.charAt(itr) || firstString.charAt(itr)==(secondString.charAt(itr)-32)
+						||firstString.charAt(itr)==secondString.charAt(itr)+32)
+				{
+					return true;
+				}
+			}
+			return false;
+	}
+		
+	}
+	public String reverse(String firstString)
+	{
+		/*
+		 * method to reverse a string
+		 * @param firstString contains string input
+		 * @param reverseString contains reverse of firstString.
+		 * @return String reverseString
+		 */
+		String reverseString="";
+		for(int itr=(firstString.length()-1); itr>=0;itr--)
+		{
+			reverseString=reverseString+firstString.charAt(itr);
+		}
+		System.out.println(reverseString);
```

Comment:
Do not put SYSO commands in logic methods. remove this even if you have added this for testing.

Reply:
Resolving as to informed her to not repeat in future.